eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/index.js

```diff
@@ -8,7 +8,6 @@ import CustomPropTypes from "custom-prop-types";
 
 import IconButtonDelete from "components/buttons/IconButtonDelete";
 import DuplicateButton from "components/buttons/DuplicateButton";
-import FadeTransition from "components/transitions/FadeTransition";
```

Comment:
Why no more fade transition on deleting questionnaire?

Reply:
Well spotted 😉 I couldn't get it to work in a reasonable way, there was a slightly breaking change and getting it to work was starting to take longer than was worth it, especially as you never really see it due to the modal being in the way.

Reply:
Okay thats fine. I think we should have a session where we decide what animations etc we should have, as I dont think we have any consistent use of them. It also needs design work to decide what the animation should do.